stubs/assertpy/assertpy/dynamic.pyi

```diff
@@ -0,0 +1,10 @@
+from collections.abc import Iterable as _Iterable
+from typing import Any
+from typing_extensions import Self, TypeAlias
+
+__tracebackhide__: bool
+
+Iterable: TypeAlias = _Iterable[Any]
```

Comment:
This is just a complicated way of importing Iterable, it doesn't need to be in the stub. (Also in some other files.)

Reply:
Done!